docs/source/guide/pt-1-intro.md

```diff
@@ -50,45 +48,73 @@ the circuit on a noisy simulator, and returns the probability of the ground
 state. See the [Executors](executors.md) section for more information on
 how to define more advanced executors.
 
+As per the noise model that is applied during execution by the simulator, 
+we choose a depolarizing channel applied to the output of each 2-qubit gates.
+Admittedly, this is not necessarily the most realistic noise,
+but for the sake of this tutorial, it is going to be useful for highlighting 
+the effect of Pauli Twirling.  
+
 ```{code-cell} ipython3
-import numpy as np
-from cirq import DensityMatrixSimulator, amplitude_damp
-from mitiq.interface import convert_to_mitiq
+from cirq import CZPowGate, CXPowGate, CircuitOperation, depolarize, DensityMatrixSimulator
+from cirq.devices.noise_model import GateSubstitutionNoiseModel
 
-def execute(circuit, noise_level=0.1):
-    """Returns Tr[ρ |0⟩⟨0|] where ρ is the state prepared by the circuit
-    executed with amplitude damping noise.
+def get_noise_model(noise_level: float) -> GateSubstitutionNoiseModel:
+    """Substitute each C-Z and C-NOT gate in the circuit 
+    with the gate itself followed by a depolarizing channel
     """
-    # Replace with code based on your frontend and backend.
-    mitiq_circuit, _ = convert_to_mitiq(circuit)
-    noisy_circuit = mitiq_circuit.with_noise(amplitude_damp(gamma=noise_level))
-    rho = DensityMatrixSimulator().simulate(noisy_circuit).final_density_matrix
-    return rho[0, 0].real
+    def noisy_c_gate(op):
+        if isinstance(op.gate, (CZPowGate, CXPowGate)):
+            return CircuitOperation(
+                Circuit(
+                    op.gate.on(*op.qubits), 
+                    depolarize(p=noise_level, n_qubits=2).on_each(op.qubits)
```

Comment:
What if you change `depolarize` to a coherent noise channel? As we can describe a depolarizing channel using [paulis](https://quantumcomputing.stackexchange.com/questions/4978/how-to-find-the-operator-sum-representation-of-the-depolarizing-channel), I don't think pauli twirling is going to do anything to tailor the noise of the input circuit.

See section 2.1.3 of https://arxiv.org/pdf/1509.02921 If a Pauli Transfer Matrix of some noise channel has off-diagonal terms, we expect the noise to be tailored after twirling.

Reply:
I did change the noise to a coherent one, please take a look.